tests/test_asset_test_endpoint.py

```
@@ -178,6 +179,35 @@ async def capture_stream(*args, **kwargs):
         assert "Reviews code for best practices" in captured_message
         assert "test this skill" in captured_message
 
+    def test_subsequent_message_sends_raw_query(self, client, mock_service, mock_asset, mock_cko_service):
+        """Test that subsequent messages (message_count > 0) send only the raw query, not asset context."""
+        mock_service.get_asset.return_value = mock_asset
+        mock_cko_service.get_or_create_asset_test_session.return_value.message_count = 1
+
+        captured_message = None
+
+        async def capture_stream(*args, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `args`

(ARG001)